.github/workflows/ci.yml

```diff
@@ -11,8 +11,10 @@ jobs:
       fail-fast: false
       matrix:
         platform:
-        - { name: Windows VS2019,       os: windows-2019 }
-        - { name: Windows VS2022,       os: windows-2022 }
+        - { name: Windows VS2019 x86,   os: windows-2019, flags: -A Win32 }
+        - { name: Windows VS2019 x64,   os: windows-2019, flags: -A x64 }
+        - { name: Windows VS2022 x86,   os: windows-2022, flags: -A Win32 }
+        - { name: Windows VS2022 x64,   os: windows-2022, flags: -A x64 }
         - { name: Windows VS2022 Clang, os: windows-2022, flags: -T ClangCL }
```

Comment:
Tiny nitpick, should the `flags` be aligned with the one below for _Linux Clang_?

Btw, YAML is really special in what it treats as strings without quotes, but if it works... 😉

Reply:
No because that one isn't aligned with the MacOS Xcode either and don't think, aligning things with "empty" lines in between makes that much sense. With auto formatters I start to not care as much about these alignments 😄

Reply:
It seems weird that we pay attention to align the 2nd column, but not the third 😅 but it's not important to bikeshed further, so let's go with the changes!